### PR TITLE
Added more detail to confirmation message for delete vocabulary

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
concerns https://github.com/mitodl/lore/issues/635

@pdpinch @noisecapella

### CODE REVIEW

File: ui/static/ui/js/utils.jsx [hunk 1]
```diff
@@ -125,16 +125,17 @@ define("utils", ["jquery", "lodash", "react", "react_infinite", "select2"],
   /**
    * Generic confirmation dialog.
    * <ConfirmationDialog
-   *       id={confirmationDialogId}
```
Comment (pdpinch) [inline ui/static/ui/js/utils.jsx:128]: What was this for and why don't we need it anymore?

**Comment by pdpinch** [comment]
Can we get a screenshot if it's not too much trouble?

File: ui/static/ui/js/utils.jsx (hunk 1 above)
Comment (noisecapella) [inline ui/static/ui/js/utils.jsx:128]: In general we don't need to set the id attribute for the confirmation dialog, we already have a reference to it and we can style it using CSS classes.

**Comment by amir-qayyum-khan** [comment]
@pdpinch  sure

screen shot 2015-09-15 at 12 50 57 pm

File: ui/static/ui/js/manage_taxonomies.jsx [hunk 2]
```diff
@@ -185,7 +185,9 @@ define('manage_taxonomies', ['react', 'lodash', 'jquery', 'utils', 'bootstrap'],
         actionButtonClass: "btn btn-danger btn-ok",
         title: "Confirm Delete",
         message: "Are you sure you want to delete vocabulary '" +
-          this.props.vocabulary.name + "'?",
+          this.props.vocabulary.name + "'? ",
+        description: "All terms in the vocabulary will be deleted and" +
+          " removed from all learning resources in the repository.",
```
Comment (pdpinch) [inline ui/static/ui/js/manage_taxonomies.jsx]: I think `in this repository` would be more clear

**Comment by pdpinch** [comment]
Thanks for the screenshot. I'm not crazy about the word wrap, but I don't think it's worth futzing with. If you can just change [the message](https://github.com/mitodl/lore/pull/649#discussion_r39505367) I think this can be merged.

**Comment by amir-qayyum-khan** [comment]
Done @pdpinch

**Comment by ShawnMilo** [comment]
This works for me locally, and the code looks reasonable. JavaScript tests are failing, though.

File: ui/static/ui/js/utils.jsx [hunk 3]
```diff
@@ -146,10 +147,16 @@ define("utils", ["jquery", "lodash", "react", "react_infinite", "select2"],
...
+      var description = '';
       if (this.props.title) {
         title = <h4 className="modal-title">{this.props.title}</h4>;
       }
 
+      if (this.props.description) {
+       description = <p className="confirmation-description">
+         {this.props.description}
+       </p>;
+      }
```
Comment (pdpinch) [inline ui/static/ui/js/utils.jsx:159]: failing tests due to indentation here: https://travis-ci.org/mitodl/lore/builds/80808186#L1059

**Comment by pdpinch** [comment]
@amir-qayyum-khan you can merge this once tests pass

**Comment by amir-qayyum-khan** [comment]
OK let me.

File: ui/static/ui/js/manage_taxonomies.jsx [hunk 4]
```diff
@@ -185,7 +185,9 @@ define('manage_taxonomies', ['react', 'lodash', 'jquery', 'utils', 'bootstrap'],
         actionButtonClass: "btn btn-danger btn-ok",
         title: "Confirm Delete",
         message: "Are you sure you want to delete vocabulary '" +
-          this.props.vocabulary.name + "'?",
+          this.props.vocabulary.name + "'? ",
+        description: "All terms in the vocabulary will be deleted and" +
+          " removed from all learning resources in this repository.",
```
Comment (pdpinch) [inline ui/static/ui/js/manage_taxonomies.jsx]: Sorry, another change to the message based on a some internal focus-grouping:

```
"Deleting this vocabulary will remove it from all learning resources."
```

A nice side-effect: no more wrapping.

**Comment by pdpinch** [comment]
@amir-qayyum-khan I'm noticing now that you have some gaps in javascript code coverage. I'd like to get this merged today, so I'm willing to let it pass, but I wanted to ask how much work it would be to cover those lines? We could open a new issue for that.

**Comment by amir-qayyum-khan** [comment]
I am rebuilding docker there was few issue sin that, once i am done i will fix it in an hour or half. Thanks

**Comment by amir-qayyum-khan** [comment]
Done with fixing coverage yesterday. you can merge it now @pdpinch

**Comment by pdpinch** [comment]
Thanks Amir. Please feel free to merge.

**Pull request merged**